examples/query/src/main.rs

```diff
@@ -31,7 +31,7 @@ fn prog() {
 
 fn map() {
     for map in query::MapInfoIter::default() {
-        println!("name={:<16} type={}", map.name, map.ty.to_string(),);
+        println!("name={:<16} type={}", map.name, map.ty,);
```

Comment:
is that trailing coma necessary? looks weird

Reply:
Rust supports trailing commas but I agree it looks unnatural. I will remove this.